articles/configure-a-404-page.mdx

```diff
@@ -0,0 +1,103 @@
+---
+_schema: default
+_uuid: 675e4636-c318-49a9-b5a4-9764180b57d9
+_created_at: '2021-09-15 23:13:40 +0000'
+title: Configure a 404 page
+nav_title: Configure a 404 page
+published: true
+image: >-
+  https://cc-dam.imgix.net/documentation/images/default-thumbnail/2023-Q4/CloudCannon-Documentation-Thumbnail.png
+description: >-
+  Configure a helpful, branded 404 page when a visitor requests a page that
+  cannot be found.
+article_category:
+  - Instructions
+article_topic:
+  - Hosting
+  - Routing
+tags:
+  - Hosting
+  - Sites
+  - Guides
+related_articles:
+related_links:
+---
+<comp.Notice info_type="important">
+  This method for configuring a 404 page is only available for Sites [hosted through CloudCannon](/documentation/articles/what-is-web-hosting/#hosting-on-cloud-cannon). If you host your Site externally, or use CloudCannon in [Headless Mode](/documentation/articles/what-is-headless-mode/), these instructions will not work.
+</comp.Notice>
+
+A "404 page" is the webpage shown to visitors when their browser cannot to find the content for the URL they requested, named after the [HTTP status code](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes) `404 Not Found`. A browser may be unable to find a page's content because:
```

Comment:
```suggestion
A "404 page" is the webpage shown to visitors when their browser cannot find the content for the URL they requested, named after the [HTTP status code](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes) `404 Not Found`. A browser may be unable to find a page's content because:
```

Small nit: _Technically_ the server is the one who cannot find the page, and is the one who provides the 404 page. The browser just shows whatever it is given. Unsure if that's an important distinction here.